root/components/home/Facts.js

```diff
@@ -0,0 +1,75 @@
+/*
+ * @flow strict
+ * Copyright (C) 2022 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+import {FontAwesomeIcon} from '@fortawesome/react-fontawesome';
+import {faMusic, faHeadphones, faCompactDisc, faUsers}
+  from '@fortawesome/free-solid-svg-icons';
+
+const Facts = (): React.Element<'section'> => (
+  <section className="p-4">
+    <div className="container">
+      <div className="row">
+        <div className="col-lg-3 col-md-6">
+          <div className="count-box p-4">
+            <FontAwesomeIcon
+              className="icon me-4"
+              icon={faMusic}
+              size="4x"
+            />
+            <div>
+              {exp.l(`<span class="fs-1 fw-bold">${1.95} M</span> Artists`)}
```

Comment:
This is not quite how it works :) See what @mwiencek proposed specifically: you'll need it to be an argument, such as:
```suggestion
              {exp.l(
                  `<span class="fs-1 fw-bold">${total_in_millions} M</span> Artists`,
                  {total_in_millions: '1.95'}
              )}
```
He proposed using `Intl.NumberFormat` too, so look into that :)

Reply:
Also, semi-relatedly, you don't need to use backticks when you only have one line (although you might end up needing several lines here, unclear).

Reply:
Make sure to remove the `$` from `${total_in_millions}` too (that still creates an expression inside backticks).  There's a couple reasons why you can't use those:
 * `${}` expressions can be dynamic (not known until runtime), but the string you pass to `l()` has to be static so that it can be looked up in the message catalog.  (The expression you have here (1.95) is constant, so the string is actually static, but the goal is to parameterize the number so that the string doesn't need to be re-translated.)
 * The template expressions can't be extracted into something usable for the translator, because there's no placeholder name for them to work with.

Reply:
Done!